Cargo.toml

```diff
@@ -104,6 +104,7 @@ members = [
     "utils/compile-time-utils",
     "utils/path",
     "utils/process",
+    "utils/std-utils",
```

Comment:
a project with such name means 'trash bin for various code that not belong anywhere' usually ;)